controllers/comments.controllers.js

```diff
@@ -0,0 +1,20 @@
+const { response } = require("../app");
+const {getArticleComments} = require("../models/comments.models")
+
+exports.getArticleComments = (req, res, next) => {
+    const {article_id} = req.params;
+    if (isNaN(article_id)){
+        return res.status(400).send({msg: "Bad Request"})
+    }
```

Comment:
This can and should be handled using a psql error and ideally handled in the model